src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -155,6 +155,8 @@
   }
 
   &.pf-m-plain {
+    justify-content: center;
```

Comment:
This declaration is in the selector below that excludes the `.pf-m-text` variation. If it applies to the `.pf-m-text` variation and you set a `width` on the options menu, you get this:

<img width="320" alt="Screen Shot 2019-08-13 at 4 46 50 PM" src="https://user-images.githubusercontent.com/35148959/62979669-0a0afd00-bdea-11e9-9801-294cfcc9e8ea.png">